airbyte-webapp/src/packages/cloud/views/settings/integrations/DbtCloudSettingsView.tsx

```diff
@@ -12,21 +12,35 @@ import styles from "./DbtCloudSettingsView.module.scss";
 
 export const DbtCloudSettingsView: React.FC = () => {
   const { mutate: submitDbtCloudIntegrationConfig, isLoading } = useSubmitDbtCloudIntegrationConfig();
+  const [hasValidationError, setHasValidationError] = useState(false);
+  const [validationMessage, setValidationMessage] = useState("");
   return (
     <SettingsCard title={<FormattedMessage id="settings.integrationSettings.dbtCloudSettings" />}>
       <Content>
         <Formik
           initialValues={{
             serviceToken: "",
           }}
-          onSubmit={({ serviceToken }) => submitDbtCloudIntegrationConfig(serviceToken)}
+          onSubmit={({ serviceToken }) => {
+            setHasValidationError(false);
+            return submitDbtCloudIntegrationConfig(serviceToken, {
+              onError: (e) => {
+                setHasValidationError(true);
+
+                setValidationMessage(e.message.replace("Internal Server Error: ", ""));
```

Comment:
[Here](https://github.com/airbytehq/airbyte-cloud/issues/3553) is the issue for the status code cleanup.